pkg/bincapz/bincapz.go

```diff
@@ -48,6 +49,10 @@ type FileReport struct {
 	PackageRisk []string `json:",omitempty" yaml:",omitempty"`
 
 	IsBincapz bool `json:",omitempty" yaml:",omitempty"`
+
+	// The original path for scanned archive files
+	// When not scanning archives, this will be empty
+	OriginalAbsPath string
```

Comment:
Add  `json:",omitempty" yaml:",omitempty"` so that the empty field doesn't show up in JSON output.

Reply:
I'm unclear on the relationship between `Path` and `OriginalAbsPath`. If `Path` is the original path requested by the user, is this just the absolute version of it?

On a related note - we should make sure that we're not storing any temporary file paths within the struct, as they aren't useful to the reader, as that file location no longer exists when they read the report.

We should however store the relative location within an archive somewhere so that they can investigate further.